flutter/test/sentry_navigator_observer_test.dart

```diff
@@ -57,7 +59,10 @@ void main() {
 
       sut.didPush(currentRoute, null);
 
-      expect(mockNativeChannel.numberOfBeginNativeFramesCalls, 1);
+      // Handle internal async method calls.
+      await Future.delayed(const Duration(milliseconds: 10), () {
+        expect(mockNativeChannel.numberOfBeginNativeFramesCalls, 1);
+      });
```

Comment:
We can likely remove this `delayed` now since we can await the call.

Reply:
Don't think we can here? `didPush` is a sync method and we can't change the signature, as we are implementing an interface. Or am i missing something here?

Reply:
Oh indeed, I missed that.